test/ng/directive/formSpec.js

```diff
@@ -420,15 +423,18 @@ describe('form', function() {
       inject(function($timeout) {
         doc = jqLite('<div>' +
                         '<form ng-submit="submitMe()">' +
-                          '<button ng-click="destroy()"></button>' +
+                          '<button type="submit" ng-click="destroy()"></button>' +
```

Comment:
No, it is not necessary. Just making it more explicit for people that might not know.